test_suite.py

```diff
@@ -30,12 +30,18 @@ class Ruby(Language):
     script_ext = ".rb"
 
 
-class JavaScript(Language):
-    name = "javascript"
+class Nodejs(Language):
+    name = "nodejs"
     interpreter = "node"
     script_ext = ".mjs"
 
 
+class Deno(Nodejs):
+    name = "deno"
+    interpreter = "deno run --allow-read"
```

Comment:
```suggestion
    interpreter = "deno run --allow-read --allow-write"
```

This will allow it to write to files